docs/components/table.md

```diff
@@ -230,6 +300,51 @@ An Example of changing the hover background might look like.
 </style>
 ```
 
+<script>
+import { defaultSorter } from '../../packages/KTable/KTable'
+export default {
+  data() {
+    return {
+      sortOrder: 'ascending',
+      sortKey: 'name',
+      defaultSorter: defaultSorter,
```

Comment:
this is a reference to a function so it works for `data`, but would prefer it to be in methods e.g.
```js
methods: {
    sortFieldHelper (key) {...},
    defaultSorter
  },

```